src/main/java/game/race/model/InputView.java

```diff
@@ -0,0 +1,55 @@
+package game.race.model;
+
+import java.util.Scanner;
+
+public class InputView {
+
+    private static final String ASK_VEHICLE_COUNT = "자동차 대수는 몇 대 인가요?";
+    private static final String ASK_TRY_COUNT = "시도할 회수는 몇 회 인가요?";
+    private static final Scanner scanner = new Scanner(System.in);
+
+    private int vehicleCnt;
+    private int tryCnt;
+
+    public int getVehicleCnt() {
+        return vehicleCnt;
+    }
+
+    public int getTryCnt() {
+        return tryCnt;
+    }
+
+    public void setVehicleCnt(int vehicleCnt) {
+        this.vehicleCnt = vehicleCnt;
+    }
+
+    public void setTryCnt(int tryCnt) {
+        this.tryCnt = tryCnt;
+    }
+
+    public void showPrompt() {
+        System.out.println(ASK_VEHICLE_COUNT);
+        String vehicles = scanner.nextLine();
+        checkInput(vehicles);
+
+        System.out.println(ASK_TRY_COUNT);
+        String trials = scanner.nextLine();
+        checkInput(trials);
+
+        vehicleCnt = Integer.parseInt(vehicles);
+        tryCnt = Integer.parseInt(trials);
+    }
```

Comment:
```suggestion
    public 요청DTO showPrompt() {
        System.out.println(ASK_VEHICLE_COUNT);
        String vehicles = scanner.nextLine();
        checkInput(vehicles);

        System.out.println(ASK_TRY_COUNT);
        String trials = scanner.nextLine();
        checkInput(trials);

        vehicleCnt = Integer.parseInt(vehicles);
        tryCnt = Integer.parseInt(trials);
        return new 요청DTO(vehicleCnt, tryCnt);
    }
```

InputView에 역할은 사용자로 부터 입력을 받는 역할이므로 
굳이 상태값을 저장하지 않아도 될 것 같아요
입력받은 값만 단순하게 전달해 줘도 되지 않을까요?

vehicleCnt, tryCnt를 멤버변수로 가지는 요청DTO를 따로 만들어 보면 될 것 같습니다.